File: src/main/java/com/gmail/borlandlp/minigamesdtools/activepoints/type/block/PrimitiveBlockPoint.java
package com.gmail.borlandlp.minigamesdtools.activepoints.type.block;

import org.bukkit.Material;

public abstract class PrimitiveBlockPoint extends StaticBlockPoint {
    private boolean hollow;
    private Material borderMaterial;
    private Material fillerMaterial;

    public Material getBorderMaterial() {
        return this.borderMaterial;
    }

    public void setBorderMaterial(Material material) {
        this.borderMaterial = material;
    }

    public Material getFillerMaterial() {
        return this.fillerMaterial;
    }

    public void setFillerMaterial(Material material) {
        this.fillerMaterial = material;
    }

    public boolean isHollow() {
        return hollow;
    }

    public void setHollow(boolean hollow) {
        this.hollow = hollow;
    }
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/activepoints/type/block/PrimitiveBlockPoint.kt
package com.gmail.borlandlp.minigamesdtools.activepoints.type.block

import org.bukkit.Material

abstract class PrimitiveBlockPoint : StaticBlockPoint() {
    var isHollow = false
    var borderMaterial: Material = Material.AIR
    var fillerMaterial: Material = Material.AIR
}